gen3-client/g3cmd/upload-manifest.go

```diff
@@ -114,12 +114,13 @@ func init() {
 	var numParallel int
 
 	var uploadManifestCmd = &cobra.Command{
-		Use:        "upload-manifest",
-		Short:      "upload files from a specified manifest",
-		Long:       `Gets a presigned URL for a file from a GUID and then uploads the specified file.`,
-		Example:    `./gen3-client upload-manifest --profile=<profile-name> --manifest=<path-to-manifest/manifest.json> --upload-path=<path-to-file-dir/>`,
-		Deprecated: `use "./gen3-client upload" instead.`,
+		Use:     "upload-manifest",
+		Short:   "upload files from a specified manifest",
+		Long:    `Gets a presigned URL for a file from a GUID and then uploads the specified file.`,
+		Example: `./gen3-client upload-manifest --profile=<profile-name> --manifest=<path-to-manifest/manifest.json> --upload-path=<path-to-file-dir/>`,
 		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println("Notice: this is the old upload flow which requires the user to provide GUIDs. If your common supports the new upload flow, consider to use \"./gen3-client upload\" instead.")
```

Comment:
Nice! I'm wondering if there are more descriptive terms for "old" and "new" upload flow, not sure if the user will understand what those are referring to?

Reply:
Right, I'm also wondering the same thing. For now even ourselves are using terms "old upload flow" and "new upload flow" and we should have some better names to differentiate them 🤔.

What do you think if we call the command to run the new upload flow a "simplified upload command"? Although it is not really that simple 🤔

Reply:
@zflamig is there a reason we should even have two flows at all? can we just get rid of the old one?

Reply:
I think we can go with those names then 👍